Dockerfile

```diff
@@ -7,9 +7,10 @@
 # for details on how to run DynamoDB locally. This Dockerfile essentially
 # replicates those instructions.
 
-# Use JDK 7. I tried alpine, but all calls to dynamo then fail silently...
-# FROM openjdk:7-jre-alpine
-FROM openjdk:7
+FROM openjdk:8-jre-alpine
```

Comment:
I could not make it work locally but it worked using `slim` tags

Image sizes:
* regular ~510 mb
* slim ~240 mb
* alpine ~120 mb

Slim will not optimize the image size but it could be already considered an improvement

Reply:
This fell off my radar, I'll take a look now and try either getting it to work reliably or else use the slim tags.

Reply:
Could you give an example of what you ran to get the above error?
I also found this which may explain why: https://github.com/uber/h3-java/issues/25

Reply:
:wave: @Jukie I was getting this kind of message

Reply:
Let's go for `slim` then, still an improvement and I love the build arg so want to get the changes in!